python_modules/dagster/dagster/_core/definitions/asset_selection.py

```diff
@@ -900,6 +900,20 @@ def __str__(self) -> str:
         return f"owner:{self.selected_owner}"
 
 
+@whitelist_for_serdes
+class CodeLocationAssetSelection(AssetSelection):
+    selected_code_location: str
+
+    def resolve_inner(
+        self, asset_graph: BaseAssetGraph, allow_missing: bool
+    ) -> AbstractSet[AssetKey]:
+        """Use this as a stub for now. We don't have a code location field on AssetNode yet."""
```

Comment:
might be worth noting in the comment what this is used for (presumably for cloud saved views), e.g. if we don't plan on this working in user code in the near future